src/pudl/transform/ferc1.py

```diff
@@ -726,6 +727,116 @@ def combine_axis_columns_xbrl(
     return df
 
 
+class MetricInputs(TransformParams):
+    """Info for testing a particular check."""
+
+    isclose_rtol: confloat(ge=0.0) = 1e-5
+    """Relative tolerance to use in :func:`np.isclose` for determining equality."""
+
+    isclose_atol: confloat(ge=0.0, le=0.01) = 1e-8
+    """Absolute tolerance to use in :func:`np.isclose` for determining equality."""
+
+
+class CalculationMetricInputs(TransformParams):
+    """Calc params organized by check type."""
+
+    error_frequency: MetricInputs = MetricInputs()
+    relative_error_magnitude: MetricInputs = MetricInputs(isclose_atol=1e-9)
+    absolute_error_magnitude: MetricInputs = MetricInputs()
+    null_calculation_frequency: MetricInputs = MetricInputs()
+    null_reported_value_frequency: MetricInputs = MetricInputs()
+
+
+class CalculationMetricTolerance(TransformParams):
+    """Tolerances for all data checks to be preformed within a grouped df."""
+
+    error_frequency: confloat(ge=0.0, le=1.0) = 0.01
+    relative_error_magnitude: confloat(ge=0.0) = 0.04
+    absolute_error_magnitude: confloat(ge=0.0) = np.inf
+    null_calculation_frequency: confloat(ge=0.0, le=1.0) = 0.7
+    """Fraction of records with non-null reported values and null calculated values."""
+    null_reported_value_frequency: confloat(ge=0.0, le=1.0) = 1.0
+    # ooof this one is just bad
```

Comment:
suggestion is to remove these default bc storing the defaults here and in `CalculationGroupChecks` and in the table params is too many place